src/map/script.c

```diff
@@ -26523,6 +26523,18 @@ static BUILDIN(rodex_sendmail)
 			return false;
 		}
 
+		if (!data_isint(script_getdata(st, param + 1))) {
+			ShowError("script:rodex_sendmail: Passed amount for item %d is not a number!\n", i + 1);
+			return false;
+		}
+
+		int amount = script_getnum(st, param + 1);
+
+		if (amount < 1 || amount > SHRT_MAX) {
```

Comment:
here should be MAX_AMOUNT

https://github.com/HerculesWS/Hercules/blob/32045249e6f91f1d09db2b63ad9eefb04fd1c1e9/src/map/pc.c#L4648-L4672

Reply:
Indeed. Fixed...